cognite_toolkit/_cdf_tk/loaders/_resource_loaders/file_loader.py

```diff
@@ -61,7 +61,6 @@
 class FileMetadataLoader(
     ResourceContainerLoader[str, FileMetadataWrite, FileMetadata, FileMetadataWriteList, FileMetadataList]
 ):
-    template_pattern = "$FILENAME"
```

Comment:
Since we are no longer using this here and only in the builder, as well as using it both for `FileMetadataLoader` and `CogniteFileLoader`. I don't think this is the correct place for it anymore.